app/javascript/packs/application/redux/points/actions.test.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { createStore, combineReducers, applyMiddleware } from "redux";
+import thunk from "redux-thunk";
+import { listPoints, createPoint } from "./actions";
+import api from "../../api";
+
+describe("listPoints()", () => {
+  it("calls the correct dispatch and returns data when successful", async () => {
+    const dispatch = jest.fn();
+    const point = { id: 1, title: "Test Point" };
+
+    jest.spyOn(api, "listPoints").mockResolvedValue({
+      ok: true,
+      data: [point]
+    });
+
+    await listPoints()(dispatch);
+
+    expect(dispatch).toHaveBeenCalledWith({
+      type: "LIST_POINTS_SUCCESS",
+      data: { [point.id]: point }
+    });
+  });
+
+  it("calls the correct dispatach when unsuccessful", async () => {
```

Comment:
Typo here that got copy pastaed to the other test, but that's not super important.  What is important is that I like these tests!

Reply:
thanaks for the catch!